<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Phosphate</family>
     <pointsize>72</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MYSQL </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCreateDB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>19</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>CREATE DATABASE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCreateTbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>19</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>CREATE TABLE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnInsertValues">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>19</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>INSERT VALUES INTO TABLE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUpdateTbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>19</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>UPDATE TABLE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAlterTbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>19</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="text">
    <string>ALTER TABLE</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelDB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>19</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>DELETE DATABASE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelTbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>19</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>DELETE TABLE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>390</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-style:italic;&quot;&gt;~&lt;/span&gt;&lt;span style=&quot; font-size:18pt; font-style:italic; text-decoration: underline;&quot;&gt;MOHAMED NAVEED&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
